daml-lf/interpreter/src/test/scala/com/digitalasset/daml/lf/speedy/RollbackTest.scala

```diff
@@ -65,10 +66,6 @@ class RollbackTest extends AnyWordSpec with Matchers with TableDrivenPropertyChe
                 u: Unit <- throw @(Update Unit) @M:MyException (M:MyException {message = "oops"});
                 x2: ContractId M:T1 <- create @M:T1 M:T1 { party = M:T1 {party} record, info = 500 }
               in upure @Unit ();
-          choice ChControllerThrow (self) (i : Unit) : Unit,
-            controllers (throw @(List Party) @M:MyException (M:MyException {message = "oops"}))
-            to
-              upure @Unit ();
```

Comment:
@nickchapman-da I drop the test, since exception thrown from the controllers cannot be caught any more. 
Is it fine, or should I change the test ?